### PR TITLE
Add fried eggs.

### PR DESCRIPTION
**Pull request by dchieh222**
#### Summary

SUMMARY: Content "Adds fried eggs and modifies fish bagel recipe accordingly"

#### Purpose of change

This adds fried eggs. This is a realistic addition for use of eggs and allows animal fat and other higher caloric items to be used in addition to eggs in other to help combat caloric usage.

#### Describe the solution

Fried eggs require food handling skill 2 and thus are fairly easy to make, they use animal fats so they can integrate more calories, and they only require a pot or any other flat surface with an ability to heat it.  

Additionally, the fish & spanish bagel recipe now can use fried eggs, in addition to its original usage of scrambled and boiled eggs. This was a logical expectation from usage of scrambled and boiled eggs.

In normal terms this recipe takes two eggs and two tbps of butter. This is one more tbps of butter than usual, but is acceptable.

#### Describe alternatives you've considered

Not add fried eggs. Drink animal fat directly and eat scrambled eggs.

#### Testing

1) Was able to use ingredients to create "fried eggs."
2) Was able to consume "fried eggs."
3) No errors encountered. 

#### Additional context

~~1) Eggs are slightly higher in caloric content than expected.~~
https://www.delish.com/cooking/recipe-ideas/a23499380/how-to-fry-an-egg/

Due to additional butter use in recipe, this actually is lower than expected caloric result.

~~2) Eggs have no batch savings - should they? Logically one could use larger quantities of eggs.~~

### CODE REVIEW

**Comment by natsirt721** [comment]
I'm not sure how batch crafting is supposed to work, but I could fry 4-5 eggs in my largest pan comfortably at the same time. More than that and I would probably want to do them serially. I guess I am opposed to the idea that they don't benefit from batch crafting at all (especially as someone who usually eats 2 or 3 eggs fried at a time) but not sure how the exact mechanism is supposed to be implemented.

**Comment by anothersimulacrum** [comment]
You've got very messy history, FYI.

File: data/json/items/comestibles/egg.json [hunk 1]
```diff
@@ -316,6 +316,21 @@
...
     "flags": [ "FREEZERBURN" ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "fried_eggs",
+    "name": { "str_sp": "fried eggs" },
+    "copy-from": "scrambled_eggs",
+    "color": "yellow",
+    "symbol": "%",
+    "quench": -1,
```
Comment (anothersimulacrum) [inline data/json/items/comestibles/egg.json:326]:
```suggestion
    "quench": -1,
    "calories": 444,
```

**Comment by dchieh222** [comment]
@natsirt721 The most accurate rate to simulate that would be the ability to fry more eggs at the same time when using a larger pan. Since there's no way to simulate that, I'll go with the idea that it has 80% batch time reduction after the first one. 

@anothersimulacrum I'll work on finding a way to get rid of all this history.

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -3466,6 +3466,21 @@
...
     "components": [ [ [ "powder_eggs", 2 ] ], [ [ "water", 2 ], [ "water_clean", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "fried_eggs",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
```
Comment (Lamandus) [inline data/json/recipes/recipe_food.json:3476]: I would lower the food handling to 1, so it is on the same level like "sloppy joes", "fried tofu" and "fried rice". Which is in my opinion harder to achieve than sunny side up.

Comment (dchieh222) [reply data/json/recipes/recipe_food.json:3476]: It should be a little more challenging than scrambled eggs imo. Ideally it should be food handling 1, but you'll fail a lot and just get scrambled eggs.

**Comment by Shodan14** [comment]
We didn't have fried eggs?

**Pull request merged**